lib/shoryuken/manager.rb

```diff
@@ -12,7 +12,17 @@ class Manager
 
     def initialize(condvar)
       @count = Shoryuken.options[:concurrency] || 25
-      raise(ArgumentError, "Concurrency value #{@count} is invalid, it needs to be a positive number") unless @count > 0
+      @fetcher_pause_interval = Shoryuken.options[:fetcher_pause_interval] || 1
+
+      unless @count > 0
+        raise(ArgumentError, "Concurrency value #{@count} is invalid, it needs to be a positive number")
+      end
+
+      if @fetcher_pause_interval < 0
+        message = "Fetcher pause interval value #{@fetcher_pause_interval} is invalid, it cannot be negative"
+        raise(ArgumentError, message)
```

Comment:
Unrelated to the PR: I'm wondering if we should move these `ArgumentError` errors to `cli.rb`, consequently failing in there, instead of later in the startup process. WDYT?